plotly_resampler/figure_resampler/figure_resampler_interface.py

```diff
@@ -206,6 +208,31 @@ def _get_current_graph(self) -> dict:
             "layout": copy(self._layout),
         }
 
+    def _parse_trace_name(
+        self, hf_trace_data: dict, slice_len: int, agg_x: np.ndarray
+    ) -> str:
+        """Parse the trace name and return the in-place updated trace dict."""
```

Comment:
return the in-place updated trace dict.

but the return type is string? 🤔

Reply:
Good catch!